### PR TITLE
Add dup check for variable labels in state paths

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add dup check for variable labels in state paths
- Fix typo in grafana.json

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/862

### CODE REVIEW

File: db/state-util.js [hunk 1]
```diff
@@ -582,6 +611,9 @@ function isValidConfigTreeRecursive(
       }
     }
     subtreePath.pop();
+    if (params.variableLabels && params.variableLabels[label] !== undefined) {
+      delete params.variableLabels[label];
```
Comment (liayoo) [inline db/state-util.js:615]: aren't we not allowing the same var labels? why do we need to delete here?

Comment (platfowner) [reply db/state-util.js:615]: oh, it corresponds to the 'pop' operation in recursion.

Comment (liayoo) [reply db/state-util.js:615]: ahhhhh yes. thanks for the explanation!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by platfowner** [comment]
Thanks for the swift review, now merging.. @liayoo

**Pull request merged**